### ISSUE TITLE
Problem building docker image for Raspberry Pi 2

### ISSUE THREAD

**Issue by jo-me** [home-assistant/appdaemon#180]
Hi,
I'm seeing an error when trying to build a docker image with the latest sources.
I've done the same thing repeatedly over the last months but apparently with the latest update of the git sources it gives an error in the last build step.
Btw, the reason I'm building my own image is that the official docker image is not compatible with the raspberry pi. The only difference is the base image which I switched to a resin raspberry python image.

```
pi@hassbian:~/docker/appdaemon $ docker build -t appdaemon_dashboard .
Sending build context to Docker daemon  13.15MB
Step 1/10 : FROM resin/raspberry-pi2-python:3.6.3
 ---> 21b337a500c3
Step 2/10 : VOLUME /conf
 ---> Running in ddf18b9f5bb8
 ---> 97f6a21993a0
Removing intermediate container ddf18b9f5bb8
Step 3/10 : VOLUME /certs
 ---> Running in 8a1eb4aefde0
 ---> 8c4b88de1972
Removing intermediate container 8a1eb4aefde0
Step 4/10 : EXPOSE 5050
 ---> Running in 5d1a4038c34a
 ---> a4db4ae5a4bb
Removing intermediate container 5d1a4038c34a
Step 5/10 : RUN mkdir -p /usr/src/app
 ---> Running in 5a5c4ac7c883
 ---> 8652e38a3cdb
Removing intermediate container 5a5c4ac7c883
Step 6/10 : WORKDIR /usr/src/app
 ---> aa24867bf038
Removing intermediate container a953b79997af
Step 7/10 : COPY . .
 ---> 4917ede5b661
Removing intermediate container 11203bcffc2e
Step 8/10 : RUN pip3 install .
 ---> Running in ad02de23a987
Processing /usr/src/app
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-jut6viej-build/setup.py", line 6, in <module>
        from appdaemon.utils import (__version__)
      File "/tmp/pip-jut6viej-build/appdaemon/utils.py", line 1, in <module>
        import requests
    ModuleNotFoundError: No module named 'requests'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-jut6viej-build/
The command '/bin/sh -c pip3 install .' returned a non-zero code: 1
pi@hassbian:~/docker/appdaemon $

```

Any ideas?

Thanks
Jochen

**Comment by marthoc** [comment]
I also ran into this problem and worked around it by changing the RUN line in step 8 to:
```
RUN pip3 install requests && pip3 install .
```

**Comment by jo-me** [comment]
Thanks, worked also for me!

**Comment by acockburn** [comment]
I'll add a section on how to buiild a raspi docker image with the input form this issue - which resin image worked for you?

**Comment by marthoc** [comment]
I used the arm32v7/python:3.6 image for RaspberryPi 3.

**Comment by acockburn** [comment]
Thanks!

There is also an automated build available which I added to the docs for 3.0.
